server/src/test/resources/configs/without-secret-size-limit.yaml

```diff
@@ -0,0 +1,157 @@
+server:
+  applicationConnectors:
+    - type: resources-https
+      port: 4445
+      keyStorePath: dev_and_test_keystore.p12
+      keyStorePassword: ponies
+      keyStoreType: PKCS12
+      trustStorePath: dev_and_test_truststore.p12
+      trustStorePassword: ponies
+      trustStoreType: PKCS12
+      wantClientAuth: true
+      validateCerts: true
+      validatePeers: true
+      crlPath: dev_and_test.crl
+      supportedProtocols: [TLSv1.2]
+      supportedCipherSuites:
```

Comment:
It would be nice if there was a way to combine keywhiz base config with a small portion of YAML that we'd like to have modified, but this is probably fine for now.

Reply:
Actually, one really cheesy way to do this would be to have the code load two files and then just merge them together, but I'll leave it as-is for now.

Reply:
Yeah, the problem is there's modest encryption and packaging (JSON, I think) overhead for our secrets, so the size of the raw secret that the user uploads doesn't exactly match what the storage layer sees. But, I thought it was better to speak in user-facing terms and put the burden of calculating a reasonable buffer on our end.